### ISSUE TITLE
[needs-docs][reports] Tweak strings for clarity, add tooltips

### ISSUE THREAD

**Issue by QGIS-feature-tracker** [qgis/QGIS-Documentation#2358]
Original commit: https://github.com/qgis/QGIS/commit/a1785e214ea58606c32d760b73bcae6d1125ca3e by nyalldawson

Rename 'Single section' to 'Static layout section'

**Comment by havatv** [comment]
We do not have a section on reports in the user manual, do we?

**Comment by DelazJ** [comment]
Right, no section yet. An issue I faced when trying to tackle the report section is that I couldn't figure out where/how i could integrate that section(/page?) to give the feature some visibility and avoid repetition. Also, we always use print layout when describing tools or process though they apply to both report and layout (and I don't know whether we have a generic word to use for both)
A renaming of [chapters](https://docs.qgis.org/testing/en/docs/user_manual/print_composer/index.html) might be needed, and maybe some moves....
